.rubocop.yml

```diff
@@ -43,6 +43,9 @@ Style/IfInsideElse:
 Style/RedundantReturn:
   Enabled: false
 
+Style/WordArray:
+  MinSize: 5
```

Comment:
What does `MinSize` do?  Is that the number of words in the array?

:rage1: :rage2: :rage3: :rage4: Why can I never find _documentation_ for Ruby code/tools?  The "configuration options" link in Rubocop's readme just points to their default configuration, in which [the `Style/WordArray` option is sadly undocumented](https://github.com/bbatsov/rubocop/blob/master/config/default.yml#L947).  The [rubydoc.info entry is equally useless](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/WordArray).  I eventually [found my answer in the source code](https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/word_array.rb#L27):

```ruby
if style == :percent && array_elems.size >= min_size
  add_offense(node, :expression, PERCENT_MSG)
end
```

So our chosen setting says:

* For arrays of 0-4 word-like elements, either style is fine.
* For arrays of 5 or more word-line elements, you must use `%w()` syntax.

Out of curiosity, why 5?